src/snapshot.ts

```diff
@@ -207,8 +144,42 @@ export class Snapshot {
   static formatName_(projectId: string, name: string) {
     return 'projects/' + projectId + '/snapshots/' + name.split('/').pop();
   }
+
+
+  create(gaxOpts?: CallOptions): Promise<CreateSnapshotResponse>;
+  create(callback: CreateSnapshotCallback): void;
+  create(gaxOpts: CallOptions, callback: CreateSnapshotCallback): void;
+  create(
+      gaxOpts?: CallOptions|CreateSnapshotCallback,
+      callback?: CreateSnapshotCallback): void|Promise<CreateSnapshotResponse> {
+    if (!(this.parent instanceof Subscription)) {
+      throw new Error(
+          `This is only available if you accessed this object through {@link Subscription#snapshot}`);
+    }
+    return (this.parent as Subscription)
+        .createSnapshot(this.name, gaxOpts! as CallOptions, callback!);
+  }
+
+  seek(gaxOpts?: CallOptions): Promise<google.pubsub.v1.SeekResponse>;
+  seek(callback: google.pubsub.v1.Subscriber.SeekCallback): void;
+  seek(
+      gaxOpts: CallOptions,
+      callback: google.pubsub.v1.Subscriber.SeekCallback): void;
+  seek(
+      gaxOpts?: CallOptions|google.pubsub.v1.Subscriber.SeekCallback,
+      callback?: google.pubsub.v1.Subscriber.SeekCallback):
+      void|Promise<google.pubsub.v1.SeekResponse> {
+    if (!(this.parent instanceof Subscription)) {
+      throw new Error(
+          `This is only available if you accessed this object through {@link Subscription#snapshot}`);
```

Comment:
Lets drop the JSDoc portion of the error message and just return `Subscription#snapshot`